src/visual.ts

```diff
@@ -1323,65 +1349,45 @@ export class DualKpi implements IVisual {
     }
 
     private static getScale(element: HTMLElement): ElementScale {
-        const clientRect: ClientRect = element.getBoundingClientRect();
+        const clientRect = element.getBoundingClientRect();
 
         return {
             x: clientRect.width / element.offsetWidth,
             y: clientRect.height / element.offsetHeight
         };
     }
 
-    private applyFontSize(element: Selection, fontSizeAutoFormattingTitle: boolean, fontSize: number) {
-        if (!fontSizeAutoFormattingTitle) {
-            element.attr("font-size", fontSize);
-        } else {
-            element.classed(this.sizeCssClass, true);
-        }
-    }
-
-    private applyBold(element: Selection, isBold: boolean) {
-        if (isBold) {
-            element.attr("font-weight", "bold");
-        } else {
-            element.attr("font-weight", null);
-        }
-    }
-
-    private applyItalic(element: Selection, isItalic: boolean) {
-        if (isItalic) {
-            element.attr("font-style", "italic");
-        } else {
-            element.attr("font-style", null);
-        }
-    }
-
-    private applyFontFamily(element: Selection, fontFamily: string) {
-        element.attr("font-family", fontFamily);
-    }
-
     private applyTextStyle(element: Selection, options: IDualKpiOptions, isTitle?: boolean) {
         const fontSizeAutoFormatting: boolean = isTitle ? options.fontSizeAutoFormattingTitle : options.fontSizeAutoFormattingValue,
             fontSize: number = isTitle ? options.titleFontSize : options.valueFontSize,
             isBold: boolean = isTitle ? options.isBoldTitle : options.isBoldValue,
             isItalic: boolean = isTitle ? options.isItalicTitle : options.isItalicValue,
+            isUnderline: boolean = isTitle ? options.isUnderlineTitle : options.isUnderlineValue,
             fontFamily: string = isTitle ? options.fontFamilyTitle : options.fontFamilyValue;
 
-        this.applyFontSize(element, fontSizeAutoFormatting, fontSize);
-        this.applyBold(element, isBold);
-        this.applyItalic(element, isItalic);
-        this.applyFontFamily(element, fontFamily);
+
+        if (fontSizeAutoFormatting) {
+            element.classed(this.sizeCssClass, true);
+        } else {
+            element.attr("font-size", fontSize);
+        }
+
+        element.attr("font-weight", isBold ? "bold" : "normal");
+        element.attr("font-style", isItalic ? "italic" : "normal");
+        element.attr("text-decoration", isUnderline ? "underline" : "normal");
```

Comment:
for a text-decoration "off" value use - "none" not "normal"

Reply:
Done.